Tests/Integration/AbstractIntegrationTest.php

```diff
@@ -150,7 +150,7 @@ protected function createAuthorizationRequest(?string $credentials, array $body
             'Authorization' => sprintf('Basic %s', base64_encode($credentials)),
         ];
 
-        return new ServerRequest([], [], null, null, 'php://temp', $headers, [], [], $body);
+        return (new ServerRequest('', '', $headers))->withParsedBody($body);
```

Comment:
Why not use `Nyholm\Psr7\Factory\Psr17Factory` to build PSR-7 instances required in tests?

Reply:
@ajgarlag Done.